library/src/jsMain/kotlin/org/eyeseetea/dhis2/lightsdk/common/ReadFileJS.kt

```diff
@@ -0,0 +1,6 @@
+package org.eyeseetea.dhis2.lightsdk.common
+
+internal actual fun readFile(file: String): String {
```

Comment:
please move this file under commonTest because there is where is used

Reply:
@xurxodev  I could not move these files to commonTest.
The file ReadFileJVM  is on androidMain package.
The file ReadFileJS is on jsMain package.
And the ReadFile is in commonMain.

When I tried to move (ReadFileJVM or ReadFileJS) the file don't compile. 
The specific js or java code can't be placed into the main package, and the IDE shows the error.

And when I tried to move the interface(ReadFile.kt) to the test package, the specific files the IDE show a declaration error because they don't have access to it from the common code.

Reply:
Ok It's my fault, I mean:
Expect code to read from the file should be under commonTest. 
Actual code for jvm to read from file should be un androidTest
Actual code for js to read from file should be un jsTest